src/components/editor/property/InputLookupQA.jsx

```diff
@@ -156,6 +157,23 @@ class InputLookupQA extends Component {
           const subauthority = lookupConfig.subauthority
           const language = lookupConfig.language
 
+          /*
+           *  There are two types of lookup: linked data and non-linked data. The API calls
+           *  for each type are different, so check the nonldLookup field in the lookup config.
+           *  If the field is not set, assume false.
+           */
+          const nonldLookup = lookupConfig.nonldLookup ? lookupConfig.nonldLookup : false
+
+          // default the API calls to their linked data values
```

Comment:
Not sure if these specific QA parameters should be hardcoded in the component but fine with leaving them here for now.

Reply:
In case it's not clear here, these specific parameters are being read in from the lookup config to signal that the URLs for this specific vocabulary will be different from the regular /search/linked_data/vocab_name path (it will be /search/vocab_name instead).  
Also, to not require resource templates to always have the property for whether or not the lookup requires the linked data or non-linked data QA paths, the code uses the regular linked data path except if the resource template indicates otherwise. https://github.com/LD4P/sinopia_editor/blob/lookup_context_api/src/components/editor/property/InputLookupQA.jsx#L172

Reply:
I can break out the context parameter check into a separate function perhaps and then test if the function picks up the parameter from the resource template correctly.

Reply:
I'm fine if you want to mark this resolved. When everything is merged, I think we could create an integration test to test for expectations arising from ShareVDE and Discogs sources.